pkg/kubectl/cmd/util/openapi/validation/validation.go

```diff
@@ -98,13 +99,13 @@ func parse(data []byte) (interface{}, error) {
 	return obj, nil
 }
 
-func getObjectKind(object interface{}) (schema.GroupVersionKind, error) {
+func getObjectKind(object interface{}) (schema.GroupVersionKind, [] error) {
 	fields := object.(map[string]interface{})
+	var listErrors []error
```

Comment:
maybe move that line somewhere else. It's kind of awkward to seperate `field := ...` and `if fields == nil {`

Reply:
Yup, makes sense!